db/migrate/20230523154812_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create the comments table
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] you should reference to author as well similar to what you did in the Post table:
```
t.references :author, null: false, foreign_key: { to_table: :users }, index: true
```